ide/eclipse/bundles/org.apache.syncope.ide.eclipse.plugin/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,48 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Plugin
+Bundle-SymbolicName: org.apache.syncope.ide.eclipse.plugin;singleton:=true
+Bundle-Version: 2.0.0.qualifier
```

Comment:
The qualifier maps to STANDALONE
Which is why the jar, the archive and the p2 site built is named "org.apache.syncope.ide.eclipse.plugin-2.0.0-SNAPSHOT"

The version number can be changed using a plugin.properties file, but this will again lead to the same problems as faced in [this issue](https://github.com/tmess567/SYNCOPE-809/issues/9). 

Also, this version number must be the same as in the pom or else it will throw the following error.
```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-packaging-plugin:0.25.0:validate-version (default-validate-version) on project org.apache.syncope.ide.eclipse.plugin: Unqualified OSGi version 1.0.0.qualifier must match unqualified Maven version 2.0.0-SNAPSHOT for SNAPSHOT builds
```
It would be better if this remains static and is changed whenever required.